### PR TITLE
add get all articles and display

### PR DESCRIPTION
**Pull request by Liam-Tomlinson**
I have done an endpoint and got it to display a list of all items on the screen.  My plan is to add screen responsiveness and styling soon.

### CODE REVIEW

File: src/componants/insideAllArticles/ArticleCard.jsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import React, { useState, useEffect } from 'react';
+
+
+const ArticlesCard = (item) => {
+
+    console.log(item.item)
```
Comment (nc-bela-bertalan) [inline src/componants/insideAllArticles/ArticleCard.jsx:6]: Please remove all debugging logs.

File: src/componants/insideAllArticles/ViewArticlesList.jsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        getArticlesList()
+        .then((results) => setarticlesList(results))
+     }, [])
+
+     console.log(articlesList[0])
+   
+    return (
+       <div>
+          {articlesList.map((item) => {
+        return <ArticlesCard item={item}/>
```
Comment (nc-bela-bertalan) [inline src/componants/insideAllArticles/ViewArticlesList.jsx:21]: Please give a unique key property to this child component inside map.

File: src/componants/insideAllArticles/ViewArticlesList.jsx [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+       </div>
+      
+       
+        )
+
+}
+
+
+// searchResults.map((item, index) => {
+//     return <ItemCard key={index} item={item}/>
```
Comment (nc-bela-bertalan) [inline src/componants/insideAllArticles/ViewArticlesList.jsx:32]: Please remove all commented out code.

File: utils/apiRequests.js [hunk 4]
```diff
@@ -0,0 +1,8 @@
+import axios from 'axios';
+
+export const getArticlesList = () => {
+    return axios.get(`https://nc-news-eph7.onrender.com/api/articles/`)
```
Comment (nc-bela-bertalan) [inline utils/apiRequests.js:4]: Great that you've extracted this functionality. I recommend using baseURL with axios 'create'.

**Review by nc-bela-bertalan — COMMENTED** [review]
Hey Liam, nice work so far, please see my comments. Also consider making your app responsive.

File: src/componants/insideAllArticles/ArticleCard.jsx (hunk 1 above)
Comment (Liam-Tomlinson) [reply src/componants/insideAllArticles/ArticleCard.jsx:6]: done

**Pull request merged**